website/Contribution/oca_module_lifecycle_maintainer_role.rst

```diff
@@ -0,0 +1,137 @@
+homogeneous content seems to be impractical, due to the difficulty to
+manage a large amount of repositories.
+
+The maintainer role has been introduced to alleviate the above issues.
+
+Being the maintainer of an addon module bears some responsibilities
+towards the community. This document aims at clarifying those.
+
+Maintainer role and responsibilities
+====================================
+
+-  Maintainers main role is to foster efficient community collaboration
+       on the addons they are responsible for. This includes
+
+   -  **Communication:** Ensuring that discussions and reviews don’t
+          stall, intervening to ping the people involved, or ask for
+          next steps whenever communication stops.
+
+   -  **Control:** Doing final checks and merging approved pull
+          requests.
+
+   -  **Versioning: **
+
+      -  Bumping version numbers during merge or immediately after.
+
+      -  For Mature addons, maintaining the recommended changelog
+             (HISTORY.rst).
+
+   -  **Support:** Handling support requests and questions.
+
+   -  **Documentation:** Documenting and managing the roadmap (future
+          improvement ideas, known issues).
+
+   -  **Guidance:** Keeping an eye on contributions in related areas to
+          help ensure functional consistency in OCA. This may include
+          helping to merge new feature in the addons they maintain, or
+          guiding contributors towards the creation of new extension
+          modules. This must not preclude the creation of addons
+          providing a competing approach, fostering innovation.
+
+-  Declared maintainers are given write access to the repository. If
+       they are not PSC member they are trusted not to do write
+       operations (eg merging pull requests) that have impact on other
+       addons than the one they maintain.
+
+-  Maintainers may proceed to some maintenance operation with relaxed
+       code review rules (eg performing simple, low risk, forward and
+       backports, maintaining documentation, applying cosmetic
+       improvements to the code such as linting). Maintainers are
+       expected to follow the highest quality standard when performing
+       such operations.
+
+-  If there is more than one declared maintainer, they must ensure good
+       coordination between them (for instance, ensure no task are left
+       unattended on the assumption that another maintainer will do it).
+
+-  PSC members should not merge PR concerning modules with declared
+       maintainers, unless the maintainer is unreactive.
+
+-  Addons with no declared maintainers are managed by the PSC.
+
+Adding, removing maintainers
+============================
+
+-  The maintainers of an addon are declared in the **maintainers** key
+       of the manifest. This key is a list of github logins. The list of
+       declared maintainers is expected to be short (one or two in
+       general).
```

Comment:
@elicoidal there are some rendering issues (see https://github.com/OCA/odoo-community.org/blob/6a293cd6d265a1383390a7360ed801a9e679898a/website/Contribution/oca_module_lifecycle_maintainer_role.rst). 

One needs to choose between bullet list and definition list.

Reply:
right: I will have a look at it!

Reply:
done